keras_cv/models/resnet_v2_backbone.py

```diff
@@ -369,35 +301,13 @@ def __init__(
         stackwise_blocks,
         stackwise_strides,
         include_rescaling,
-        include_top,
```

Comment:
for boolean parameters such as include_rescaling I think a * to make keyword required helps readability